config/ompi_check_libfca.m4

```diff
@@ -25,20 +25,17 @@ AC_DEFUN([OMPI_CHECK_FCA],[
     OPAL_CHECK_WITHDIR([fca], [$with_fca], [lib/libfca.so])
 
     AS_IF([test "$with_fca" != "no"],
-          [AS_IF([test ! -z "$with_fca" && test "$with_fca" != "yes"],
+          [ompi_check_fca_libs=fca
+           AS_IF([test ! -z "$with_fca" && test "$with_fca" != "yes"],
 			  [ompi_check_fca_dir=$with_fca
```

Comment:
`$with_fca` might equal `no` at this point (if the user specified `--without-fca`).